cmd/tle/commands/commands.go

```diff
@@ -124,9 +131,15 @@ func parseCmdline(f *Flags) {
 	flag.StringVar(&f.Duration, "D", f.Duration, "how long to wait before being able to decrypt")
 	flag.StringVar(&f.Duration, "duration", f.Duration, "how long to wait before being able to decrypt")
 
+	flag.StringVar(&f.Time, "T", f.Time, "a UTC time value in RFC3339 format")
+	flag.StringVar(&f.Time, "time", f.Time, "a UTC time value in RFC3339 format")
+
 	flag.StringVar(&f.Output, "o", f.Output, "the path to the output file")
 	flag.StringVar(&f.Output, "output", f.Output, "the path to the output file")
 
+	flag.StringVar(&f.RawInput, "I", f.RawInput, "raw input to be encrypted")
```

Comment:
As Thibault suggested in slack, I think it would be more canonical to pass data to stdin rather than pass an extra flag

Reply:
I think we should still have a flag.. I am trying to use this binary in android and ios. Passing input to stdin might be tricky there ?